doc/websocket.qbk

```diff
@@ -227,7 +227,7 @@ void echo(beast::websocket::stream<boost::asio::ip::tcp::socket>& ws)
 {
     beast::streambuf sb;
     beast::websocket::opcode::value op;
-    ws.read(sb);
+    ws.read(op, sb);
```

Comment:
Glad to hear it! The docs are still a work in progress so if you need anything in particular explained, just open up an issue and I'll get to it promptly